### ISSUE TITLE
Help with shutter control please

### ISSUE THREAD

**Issue by pipa2493** [snowdd1/homebridge-knx#133]
Hello. 
please tell me what design should be used, and it is already confused...
I have no feedback from the drives 
2/0/0-telegram 0 (close), telegram 1 (open)
2/0/1-stop
![default](https://user-images.githubusercontent.com/45736963/49887942-c3c26000-fe4e-11e8-9f2a-12bab5dac0bd.jpg)

**Comment by snowdd1** [comment]
What is the third line in the picture?

**Comment by pipa2493** [comment]
1 byte is the value for the position of the curtains, but do not use it because there is no feedback from the motor

**Comment by pipa2493** [comment]
use 1 bit to control.
Is it possible in homekit?

**Comment by snowdd1** [comment]
No. You need to specify the run times in ETS for your motor, so the actuator control can calculate run times to percentages.

**Comment by pipa2493** [comment]
Ok. Thanks. What about feedback?
